cmd/telegraf/cmd_plugins.go

```diff
@@ -3,6 +3,8 @@ package main
 
 import (
 	"fmt"
+	"github.com/influxdata/telegraf/plugins/parsers"
+	"github.com/influxdata/telegraf/plugins/serializers"
```

Comment:
Can you move these down to the other telegraf inputs? We like to keep the stdlib imports on their own.